packages/overlay/overlay-trigger.md

```diff
@@ -49,8 +49,8 @@ Here a default `<overlay-trigger>` manages content that is triggered by click an
     }
 
     .tooltip {
-        background-color: var(--spectrum-global-color-gray-800);
-        color: var(--spectrum-global-color-gray-50);
+        background-color: var(--spectrum-gray-800);
```

Comment:
Not sure this rule set is being used in the demo?

Reply:
Removed!